src/shared/components/customise-table/CustomiseTable.tsx

```diff
@@ -40,12 +35,9 @@ const CustomiseTable: FC<CustomiseTableProps> = ({
       className="customise-table"
       data-testid="customise-table-form"
     >
-      <ColumnSelect onChange={handleChange} selectedColumns={selectedColumns} />
+      <ColumnSelect onChange={setColumns} selectedColumns={columns} />
       <div className="button-group sticky-bottom-right sliding-panel__button-row">
-        <Button variant="secondary" type="button" onClick={handleCancel}>
-          Cancel
-        </Button>
-        <Button type="submit">Save</Button>
+        <Button type="submit">Close</Button>
```

Comment:
It's neat to see the real-time updates but have a few reservations:
1. Will this create too many unnecessary requests?
2. Will users miss the ability to revert to what they had before (not just to default)? I guess user testing will only be able to answer this.

Reply:
Yeah, I wasn't sure about this behaviour either, hence the draft. I thought it could make sense now that we see both the table and the sidepanel at the same time, but the 2 points you raised are tradeoffs indeed